tool/dumper/vkgcPipelineDumper.cpp

```diff
@@ -189,6 +189,22 @@ uint64_t VKAPI_CALL IPipelineDumper::GetShaderHash(const void *moduleData) {
   return MetroHash::compact64(reinterpret_cast<const MetroHash::Hash *>(&shaderModuleData->hash));
 }
 
+// =====================================================================================================================
+// Calculates graphics shader binary hash code.
+//
+// @param pPipelineInfo : Info to build this partial graphics pipeline
+// @param stage : The shader stage for which the code is calculated
+uint64_t VKAPI_CALL IPipelineDumper::GetGraphicsShaderBinaryHash(const GraphicsPipelineBuildInfo *pipelineInfo,
```

Comment:
I am considering if we can completely follow LLPC coding style in such interface files used by driver. The origin coding style transition left this problem because it needs touch driver codes to make those changes. Note sure if we decide to resolve it now. Of course, this issue is unrelated with this change.